Moodis/Moodis/Feature/Statistics/CalendarForm.cs

```diff
@@ -11,12 +11,14 @@ public partial class CalendarForm : Form
         private CalendarViewModel calendarViewModel;
         private Form parentForm;
         private const string FormatDouble = "N3";
+        private string username;
 
-        public CalendarForm(CalendarViewModel viewModel, Form form)
+        public CalendarForm(CalendarViewModel viewModel, Form form, string username)
         {
             InitializeComponent();
             calendarViewModel = viewModel;
             parentForm = form;
+            this.username = username;
```

Comment:
isn't it possible to get the username from static signInViewModel current user?

Reply:
We need different usernames, so we can see different users statistics, so we cant use local username